### ISSUE TITLE
Read from dth11 or dth22

### ISSUE THREAD

**Issue by rainmakerho** [selomkofori/ds18b20_win10iot#1]
Hi, 
   Is possible use this way to read temperature from DTH11 or DTH22 ?
Thanks.

**Comment by selomkofori** [comment]
No, the DHT22 uses a proprietary 1-wire interface, however there is already code out there to read the dht11/22 from microsofts iot samples. https://ms-iot.github.io/content/en-US/win10/samples/GpioOneWire.htm

**Comment by rainmakerho** [comment]
Thanks.
But microsoft's sample is C++ not c#.
